content/en/account_management/billing/usage_attribution.md

```diff
@@ -76,6 +76,12 @@ Monthly data can also be pulled using the tool's public API. For more informatio
 
 ### Daily usage attribution
 
+{{< site-region region="us,eu" >}}
+<div class="alert alert-info">
+Daily usage attribution is only available for organizations in a US1 or EU1 datacenter.
+</div>
+
+
 This section provides daily reports at an hourly granularity to dig into time frames. It also provides a concatenation of all reports during a given month.
 
 - Clicking on a specific time period expands a view on the right where reports can be downloaded as a `.tsv`.
```

Comment:
Is .tsv correct. Or should it be `.csv`? And if you go with the monospace file extension wording rather than the acronym, add "file" after: `downloaded as a .csv file` or `downloaded as a CSV.`

Reply:
Oh wow, I completely missed this paragraph in the original doc. I'll download one and check

Reply:
@kayayarai `.tsv` is correct